test/config/TestConfig.cpp

```diff
@@ -211,7 +211,12 @@ TEST_F(ConfigTestFixture, AllFeaturesEnabled)
     config.sampleShadow.SerializeToObject(sampleShadow);
     ASSERT_STREQ("shadow-name", sampleShadow.View().GetString(config.sampleShadow.JSON_SAMPLE_SHADOW_NAME).c_str());
     ASSERT_STREQ("", sampleShadow.View().GetString(config.sampleShadow.JSON_SAMPLE_SHADOW_INPUT_FILE).c_str());
-    ASSERT_STREQ("", sampleShadow.View().GetString(config.sampleShadow.JSON_SAMPLE_SHADOW_OUTPUT_FILE).c_str());
+    ostringstream outputPathStream;
+    outputPathStream << Config::DEFAULT_SAMPLE_SHADOW_OUTPUT_DIR << Config::DEFAULT_SAMPLE_SHADOW_DOCUMENT_FILE;
```

Comment:
DRY. Make `outputPath` a member variable of the `ConfigTestFixture`, initialize and extract path in `ConfigTestFixture::SetUp()`, and then you can reference it inside in any test that uses the fixture (tests with `TEST_F` prefix).